scripts/st2snpseqpacks-startup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+## Install dependencies
+apt-get update && apt-get install -y gcc curl vim python
```

Comment:
At least `gcc` needs to be installed in the st2actionrunner container as well. 
The `st2 run packs.setup_virtualenv packs=snpseq_packs` part of the startup script fails for me since this package is missing.

Reply:
Sorry about that, should be fixed now!